<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createpartition</class>
 <widget class="QWidget" name="createpartition">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>create a new volume</string>
  </property>
  <widget class="QLineEdit" name="lineEditVolumePath">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolumePath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>path to volume</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCreate">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>c&amp;reate</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;cancel</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassphrase1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassPhrase">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>passphrase</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassPhrase2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelRepeatPassPhrase">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>repeat passphrase</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOpenKeyFile">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>open a key file</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="rbPassphrase">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>95</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>passphrase</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPassphraseFromFile">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>passphrase from a key file</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelfs">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>file system type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxFS">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true">ext4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">ext3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">vfat</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelvolumetype">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>volume type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxVolumeType">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string notr="true">luks</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">plain</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelrng">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>rng</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRNG">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true">/dev/urandom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">/dev/random</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditVolumePath</tabstop>
  <tabstop>rbPassphrase</tabstop>
  <tabstop>rbPassphraseFromFile</tabstop>
  <tabstop>lineEditPassphrase1</tabstop>
  <tabstop>pbOpenKeyFile</tabstop>
  <tabstop>lineEditPassPhrase2</tabstop>
  <tabstop>comboBoxFS</tabstop>
  <tabstop>comboBoxVolumeType</tabstop>
  <tabstop>comboBoxRNG</tabstop>
  <tabstop>pbCreate</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
